Doc/library/sqlite3.rst

```diff
@@ -413,6 +413,17 @@ Connection Objects
 
 .. class:: Connection
 
+   Each open SQLite database is represented by a connection object.
+   A connection object is created with :func:`sqlite3.connect`.
+   The main purpose of connection objects is creating
+   :class:`cursors objects <Cursor>`,
```

Comment:
```suggestion
   :class:`Cursor` objects
```

Avoid grammar issues and whole expression being made (unintentionally) literal, and use literal class name